build/script/package.sh

```diff
@@ -33,35 +31,34 @@ then
   cp adapter/adapter-json/target/*-with-dependencies.jar ./release/adapter/adapter_json.jar
   cp adapter/adapter-postgis/target/*-with-dependencies.jar ./release/adapter/adapter_postgis.jar
   cp backend/target/backend*.jar ./release/bin/backend.jar
+  echo "build core modules done..."
+}
+
+if [ $# -eq 0 ]; then
+  buildFrontEnd
+  buildCoreModule
+elif [ $1 == "core" ]; then
+  buildCoreModule
 else
-  if [ $1 == "backend" ]
-  then
+  if [ $1 == "backend" ]; then
     buildFrontEnd
-    mvn install -Pdocker -T 0.5C -Dmaven.test.skip=true -pl $1
-    cp backend/target/backend*.jar ./release/bin/backend.jar
-    return
   fi
   mvn install -Pdocker -T 0.5C -Dmaven.test.skip=true -pl $1
```

Comment:
The terminal might go wrong when we want to compile module of adapter-*, because adapter-* is the sub-module of adapter. 
For example, 'mvn install -Pdocker -T 0.5C -Dmaven.test.skip=true -pl adapter-postgresql' is wrong.